zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -581,24 +609,11 @@ impl<F: Field> SignVerifyChip<F> {
             )?
         };
 
-        let pk_hash_rlc = self.assign_rlc_le(
-            config,
-            ctx,
-            chips,
-            "pk_hash_rlc",
-            config.q_rlc_evm_word,
-            challenges.evm_word(),
-            iter::empty()
-                .chain(pk_hash_lo.into_iter().rev())
-                .chain(pk_hash_hi.into_iter().rev().map(Term::unassigned)),
-        )?;
-
-        self.enable_keccak_lookup(config, ctx, &is_address_zero, &pk_rlc, &pk_hash_rlc)?;
-
-        Ok(AssignedSignatureVerify {
-            address,
-            msg_hash_rlc,
-        })
+        // pk_hash is big-endian, need to covert to little-endian
+        pk_hash.reverse();
+        let pk_hash_word = self.assign_word(config, ctx, pk_hash)?;
```

Comment:
Following previous comment:

Replace this by something like:
```rust
let pk_hash_word = Word::new([address.lo(), address.hi() + int_be(pk_hash[..12]) * 2^12]);
```
You may need to do some of these operations with the `main_gate`.  The main point is that the `pk_hash_word` is constrained to be built with the 20 bytes of the `address`.

Reply:
fixed in [11db585](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1418/commits/11db585e0bae6b8da05c3575897b13389d9d7d8a)